src/encoded/static/libs/bootstrap/modal.js

```diff
@@ -252,6 +251,8 @@ ModalFooter.propTypes = {
         PropTypes.object, // Submit button is a React component; just render it
         PropTypes.func, // Function to call when default-rendered Submit button clicked
     ]),
+    submitTitle: PropTypes.string, // Title for default submit button
+    addCss: PropTypes.string, // CSS classes to add to modal header
```

Comment:
I hate making the assumption of what someone would and would not expect, and just try to always add a comment to each property so that anyone reading it would know what it is without needing context. But yes, it should be footer. Should I change that? I once corrected a clearly incorrect comment and you said, “Unneeded change!”

Reply:
Yes please do change it since we needed to edit the line of code for the feature.

Reply:
OK got it. Thanks!